### PR TITLE
Added example for social network

### PR DESCRIPTION
**Pull request by eduardonunesp**

### CODE REVIEW

File: docs/simple-social-network-example.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+### 2.) Clone the example project
+
+```bash
+git clone https://github.com/loomnetwork/solidity-social-example
+```
+
+### 3.) Start the DappChain
+
+```bash
+cd web3-loom-provider
```
Comment (enlight) [inline docs/simple-social-network-example.md]: Where did `web3-loom-provider` come from?

File: docs/simple-social-network-example.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+brew install redis
+
+# Start services on macOS
+brew services start elasticsearch
+brew services start redis
+```
+
+### 5.) Start indexer
+
+The `indexer.js` is a service which will receive all events from the smart contract and feeds a cache layer built on a message queue and a fast database (Redis + ElasticSearch), it also serves the app with a fresh data on the address `http://localhost:8081/posts` or `http://localhost:8081/comments`
```
Comment (enlight) [inline docs/simple-social-network-example.md]:
```
The indexer is a service that will receive all events from the smart contract and feed a cache layer built on a message queue and a fast database (Redis + ElasticSearch).
The data accumulated by this service is served at `http://localhost:8081/posts` and `http://localhost:8081/comments`
```

File: docs/simple-social-network-example.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+```bash
+# On second terminal
+cd web3-loom-provider/webclient
+node indexer.js
+```
+
+### 6.) Start the web server
+
+Finally we're going to start the webserver which will supply the interface allowing users to interact to the smart contracts on the Loom DappChain
```
Comment (enlight) [inline docs/simple-social-network-example.md]:
```
The webserver will serve the frontend which allows users to interact with the smart contracts on the Loom DappChain.
```

File: docs/simple-social-network-example.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+# Simple Social Network Example
+
+Sample DAppChain showing the integration between LoomProvider, Web3 and [Loom.js](https://github.com/loomnetwork/loom-js) for a super simple social network interaction, where users can post and comment on posts. Source is on [Github Websocket Demo](https://github.com/loomnetwork/solidity-social-example)
+
+![](https://dzwonsemrish7.cloudfront.net/items/2W3c2O3G2A1q1l3f3D3d/Screen%20Recording%202018-05-29%20at%2003.35%20PM.gif)
+
+> The smart contract compiled for this example uses the Solidity version 0.4.24 (SimpleSocialNetwork.sol)
+
+> This example also uses Redis and ElasticSearch to manages a cache layer
```
Comment (enlight) [inline docs/simple-social-network-example.md]: Lets remove this line, your mention Redis & ElasticSearch later on anyway.

File: docs/simple-social-network-example.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+Development
+----
+
+### 1.) Run your own DappChain
+
+Please consult the [Loom SDK docs](https://loomx.io/developers/docs/en/prereqs.html) for further instruction on running your own DappChain.
```
Comment (enlight) [inline docs/simple-social-network-example.md]:
```
Please ensure you've installed Golang as documented in the [prerequisites](https://loomx.io/developers/docs/en/prereqs.html).
```
And I think you can just merge sections 1, 2, and 3 together,

File: docs/simple-social-network-example.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
+---
+id: simple-social-network-example
+title: Simple Social Network Example
+sidebar_label: Simple Social Network Example
+---
+
+# Simple Social Network Example
+
+Sample DAppChain showing the integration between LoomProvider, Web3 and [Loom.js](https://github.com/loomnetwork/loom-js) for a super simple social network interaction, where users can post and comment on posts. Source is on [Github Websocket Demo](https://github.com/loomnetwork/solidity-social-example)
```
Comment (enlight) [inline docs/simple-social-network-example.md]:
```
This example shows how to build a social network on a Loom DAppChain, using `Web3` and [Loom.js](https://github.com/loomnetwork/loom-js).
In this simple social network users can post and comment on posts. Source is available on [Github](https://github.com/loomnetwork/solidity-social-example).
```

File: docs/simple-social-network-example.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+yarn run proto
+
+# Start the demo
+yarn start
+
+```
+
+### 7.) Running
+
+The Simple Social Network web interface will be available on `http://localhost:8080`
```
Comment (enlight) [inline docs/simple-social-network-example.md]:
```
Open `http://localhost:8080` in your web browser to use the Simple Social Network.
```

File: docs/simple-social-network-example.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+.
+├── README.md
+├── dappchain
+│   ├── SimpleSocialNetwork.sol
+│   ├── contracts
+│   │   ├── SimpleSocialNetwork.abi
+│   │   └── SimpleSocialNetwork.bin
+│   ├── genesis.example.json
+│   └── genesis.json
+└── webclient
```
Comment (mattkanwisher) [inline docs/simple-social-network-example.md]: lets not put project structure into the docs unless there is some relevance. Like we explain where the files are

File: docs/simple-social-network-example.md [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+    │   ├── index.js
+    │   ├── indexer_client.js
+    │   ├── post.js
+    │   └── text.js
+    ├── webpack.config.js
+    └── yarn.lock
+```
+
+
+Loom Network
```
Comment (mattkanwisher) [inline docs/simple-social-network-example.md]: we shouldn't blanket copy and paste from readme, things like this license and loom network at the bottom

**Pull request merged**